src/constants.js

```diff
@@ -22,6 +22,9 @@ export const ITEM_TYPE_DRUG = "D";
 export const ITEM_TYPE_MEDICAL_CONSUMABLE = "M";
 export const ITEM_TYPES = [ITEM_TYPE_DRUG, ITEM_TYPE_MEDICAL_CONSUMABLE];
 
+export const SERVICE_CODE_MAX_LENGTH = 6;
+export const ITEM_CODE_MAX_LENGTH = 6;
```

Comment:
Can these constants be modified from configuration via Django Admin?

Reply:
@dragos-dobre I did some research and I discussed with Kamil and Damian about this issue. From our perspective, it would be a better idea to create a seperated ticket where we would take care of configurations across the whole app because it should be improved not only here. Let me know what do you think about this approach.